docs/frontend_development.md

```diff
@@ -84,3 +84,17 @@ $ hass --skip-pip
 ```
 
 [hass-polymer]: https://github.com/home-assistant/home-assistant-polymer
+
+## Troubleshooting
```

Comment:
I think that we might just incorporate this into development. Everyone will be hit by this.